Automattic/simplenote-android
# Switch to a shared debug keystore

### Fix
This PR updates the build configuration in order to make debug builds use a shared keystore instead of the master key.

### Test
1. Checkout this branch.
2. Pull the latest mobile secrets. 
3. Run `./gradlew installDeps` to update your local configuration.
4. Build Simplenote debug and verify it works as expected.
5. Download the Installable build from this PR and verify it works as expected. 

### Review
Only one developer is required to review these changes, but anyone can perform the review.

**Note**: When the review is completed, before merging this PR, the test branch in the mobile repository shall be merged and the configuration in this PR updated.

### Release
These changes do not require release notes.
